commands/partner_interconnect_attachment.go

```diff
@@ -153,3 +203,169 @@ func RunPartnerInterconnectAttachmentCreate(c *CmdConfig) error {
 }
+
+// RunPartnerInterconnectAttachmentGet retrieves an existing Partner Interconnect Attachment by its identifier.
+func RunPartnerInterconnectAttachmentGet(c *CmdConfig) error {
+
+	if err := ensurePartnerAttachmentType(c); err != nil {
+		return err
+	}
+
+	err := ensureOneArg(c)
+	if err != nil {
+		return err
+	}
+	iaID := c.Args[0]
+
+	pias := c.PartnerInterconnectAttachments()
+	interconnectAttachment, err := pias.GetPartnerInterconnectAttachment(iaID)
+	if err != nil {
+		return err
+	}
+
+	item := &displayers.PartnerInterconnectAttachment{
+		PartnerInterconnectAttachments: do.PartnerInterconnectAttachments{*interconnectAttachment},
+	}
+	return c.Display(item)
+}
+
+// RunPartnerInterconnectAttachmentList lists Partner Interconnect Attachment
+func RunPartnerInterconnectAttachmentList(c *CmdConfig) error {
+
+	if err := ensurePartnerAttachmentType(c); err != nil {
+		return err
+	}
+
+	pias := c.PartnerInterconnectAttachments()
+	list, err := pias.ListPartnerInterconnectAttachments()
+	if err != nil {
+		return err
+	}
+
+	item := &displayers.PartnerInterconnectAttachment{PartnerInterconnectAttachments: list}
+	return c.Display(item)
+}
+
+// RunPartnerInterconnectAttachmentUpdate updates an existing Partner Interconnect Attachment with new configuration.
+func RunPartnerInterconnectAttachmentUpdate(c *CmdConfig) error {
+	if err := ensurePartnerAttachmentType(c); err != nil {
+		return err
+	}
+
+	err := ensureOneArg(c)
+	if err != nil {
+		return err
+	}
+	peeringID := c.Args[0]
+
+	r := new(godo.PartnerInterconnectAttachmentUpdateRequest)
+	name, err := c.Doit.GetString(c.NS, doctl.ArgPartnerInterconnectAttachmentName)
+	if err != nil {
+		return err
+	}
+	r.Name = name
+
+	vpcIDs, err := c.Doit.GetString(c.NS, doctl.ArgPartnerInterconnectAttachmentVPCIDs)
+	if err != nil {
+		return err
+	}
+	r.VPCIDs = strings.Split(vpcIDs, ",")
+
+	interconnectAttachment, err := c.PartnerInterconnectAttachments().UpdatePartnerInterconnectAttachment(peeringID, r)
+	if err != nil {
+		return err
+	}
+
+	item := &displayers.PartnerInterconnectAttachment{
+		PartnerInterconnectAttachments: do.PartnerInterconnectAttachments{*interconnectAttachment},
+	}
+	return c.Display(item)
+}
+
+// RunPartnerInterconnectAttachmentDelete deletes an existing Partner Interconnect Attachment by its identifier.
+func RunPartnerInterconnectAttachmentDelete(c *CmdConfig) error {
+
+	if err := ensurePartnerAttachmentType(c); err != nil {
+		return err
+	}
+
+	err := ensureOneArg(c)
+	if err != nil {
+		return err
+	}
+	iaID := c.Args[0]
+
+	force, err := c.Doit.GetBool(c.NS, doctl.ArgForce)
+	if err != nil {
+		return err
+	}
+
+	if force || AskForConfirmDelete("Partner Interconnect Attachment", 1) == nil {
+
+		pias := c.PartnerInterconnectAttachments()
+		err := pias.DeletePartnerInterconnectAttachment(iaID)
+		if err != nil {
+			return err
+		}
+
+		wait, err := c.Doit.GetBool(c.NS, doctl.ArgCommandWait)
+		if err != nil {
+			return err
+		}
```

Comment:
The `--wait` flag isn't defined on the delete command, so this doesn't do anything. Running `doctl network interconnect-attachment delete --wait f81d4fae-7dec-11d0-a765-00a0c91e6bf6` produces the error

```
Error: unknown flag: --wait
```